bin/ColorScript/Sources/ColorScriptCore/Sources/ColorScriptCore.swift

```diff
@@ -0,0 +1,118 @@
+// swiftlint:disable force_unwrapping
+import Foundation
+
+extension Dictionary {
+  public func withAllValuesFrom(_ other: Dictionary) -> Dictionary {
+    var result = self
+    other.forEach { result[$0] = $1 }
+    return result
+  }
+}
+
+public struct Color {
+
+  public let data: Data
+
+  public init(data: Data) {
+    self.data = data
+  }
+
+  public func colors() -> [(key: String, value: String)]? {
+    do {
+      if let json = try (JSONSerialization.jsonObject(with: data, options: []) as? [String: String]) {
+        let colors = json
+          .map { (key: $0, value: $1) }
+          .sorted { $0.key < $1.key }
+        return colors
+      }
+      return nil
```

Comment:
I think it might be more useful for us to throw an error here instead of returning `nil`, which could make it harder to figure out what happened that caused the script to crash further down (when we unwrap the output `colors()!`)